contracts/contracts/core/PreconfManager.sol

```diff
@@ -197,35 +196,33 @@ contract PreconfManager is
     /**
      * @dev Open a commitment
      * @param unopenedCommitmentIndex The index of the unopened commitment
-     * @param bid The bid amount
+     * @param bidAmt The bid amount
      * @param blockNumber The block number
      * @param txnHash The transaction hash
      * @param revertingTxHashes The reverting transaction hashes
      * @param decayStartTimeStamp The start time of the decay
      * @param decayEndTimeStamp The end time of the decay
      * @param bidSignature The signature of the bid
-     * @param commitmentSignature The signature of the commitment
      * @param sharedSecretKey The shared secret key
      * @return commitmentIndex The index of the stored commitment
      */
     function openCommitment(
```

Comment:
Shouldn't we also be passing in the chainID?

Reply:
No need, we already have it in domain separator